manifests/modular-architecture/deployment.yaml

```
@@ -0,0 +1,53 @@
+    resources:
+      limits:
+        cpu: 500m
+        memory: 1Gi
+      requests:
+        cpu: 500m
+        memory: 1Gi
+    ports:
+      - containerPort: 8043
+        name: https
+    volumeMounts:
+    - name: proxy-tls
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)